RELEASE-NOTES.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## v1.11.0 - 2019-01-15
+Includes performance improvements that will allow syncing to start sooner on client
+and for onSync done to resolve more quickly.
+
+* feat: feat: cache has entries and send on message
```

Comment:
typo, "feat:" twice :)

Reply:
Other than that, this should be good for release :)

Reply:
will fix that 👍